docs/src/components/MDXContent/MDXContent.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow strict */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { MDXProvider } from '@mdx-js/react';
+import { cssModules } from 'bpk-react-utils';
+import BpkContentContainer from 'bpk-component-content-container';
+
+import BpkMarkdownRenderer from '../DocsPageBuilder/BpkMarkdownRenderer';
+
+import STYLES from './MDXContent.scss';
+import Renderer from './renderers';
+
+const getClassName = cssModules(STYLES);
+
+type Props = {
+  Content: any,
+  darkBackground: boolean,
```

Comment:
I just checked the codebase and BpkMarkdownRenderer's `darkBackground` prop isn't used anywhere, so we shouldn't port it to this component.

Reply:
Its used in the renderer for the table so needs to be passed down to it for it to be used.

You can see it being used here: https://github.com/Skyscanner/backpack-docs/blob/master/docs/src/components/DocsPageBuilder/DocsPageBuilder.js#L224

Reply:
So I guess we will need it when we migrate other pages over?

Reply:
Huh, I must have missed that when I checked the codebase, apologies!

Technically it's still not used, because we moved away from `DocsPageBuilder` for all component pages and it only still exists because of some token pages that use it, and I'm confident that there are no pages that use a readme (and therefore the dark background) left.

I think we should remove it, it's yet another extra complication we shouldn't have added in the first place, so it'd be good to not have it in our new shiny world